### ISSUE TITLE
Autocompletion for including headers not working

### ISSUE THREAD

**Issue by lukehayes** [lvzixun/clang-complete#51]
I have gotten the autocompletion to work but am finding that when I go to include a header there is no autocompletion for potential headers.

Is there meant to be? If so, what am I doing wrong?

If not, please close the issue :)

**Comment by blmhemu** [comment]
Yeah , I too have that problem. One thing I noticed was that if you didn't include the headers the corresponding functions will NOT be suggested.
